jetbrains-core/tst/software/aws/toolkits/jetbrains/services/lambda/python/PythonLambdaHandlerResolverTest.kt

```diff
@@ -106,96 +115,92 @@ class PythonLambdaHandlerResolverTest {
         val virtualFile = createHandler("src/hello_world/foo_bar/app.py")
         createInitPy("src/hello_world/foo_bar")
         createInitPy("src/hello_world")
+        createRequirementsTxt("src")
 
         markAsSourceRoot(virtualFile.parent.parent) // hello_world
 
         assertHandler("hello_world.foo_bar.app.handle", true)
     }
 
-    @Test
-    fun findDoesntWorkIfNotASourceOrContentRoot() {
-        createHandler("src/hello_world/foo_bar/app.py")
-        createInitPy("src/hello_world")
-        createInitPy("src/hello_world/foo_bar")
-
-        assertHandler("hello_world.foo_bar.app.handle", false)
-    }
-
     @Test
     fun findDoesntWorkIfParentFolderDoesntExist() {
         createHandler("src/hello_world/foo_bar/app.py")
         createInitPy("src/hello_world/foo_bar")
+        createRequirementsTxt(".")
 
         assertHandler("doesnt_exist/foo_bar.app.handle", false)
     }
 
     @Test
     fun invalidHandlerReturnsNothing() {
         createHandler("hello_world/app.py")
+        createRequirementsTxt(".")
 
         assertHandler("doesnt_exist", false)
     }
 
     @Test
     fun findWorksByTopLevelModule() {
         createHandler("app.py")
+        createRequirementsTxt(".")
 
         assertHandler("app.handle", true)
     }
 
-    @Test
-    fun findWorkIfRequirementsFileIsFound() {
-        createHandler("src/hello_world/foo_bar/app.py")
-        createInitPy("src/hello_world")
-        createInitPy("src/hello_world/foo_bar")
-        projectRule.fixture.addFileToModule(projectRule.module, "src/requirements.txt", "")
-
-        assertHandler("hello_world.foo_bar.app.handle", true)
```

Comment:
why isn't this one valid?

Reply:
its duplicated by all the other testss since all the other valid tests require the file now